airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -416,6 +519,12 @@ def transform(self, record: Mapping[str, Any]) -> Mapping[str, Any]:
 
 
 class Issues(IncrementalGithubStream):
+    """
+    API docs: https://docs.github.com/en/rest/reference/issues#list-repository-issues
+    """
+
+    page_size = 50  # `issues` is a large stream so it's better to set smaller page size.
```

Comment:
shouldn't this also have pk = `number`?

Reply:
checking..

Reply:
I think number is not unique across repos, id should be enough